app/controllers/api/base_controller/parameters.rb

```diff
@@ -93,8 +93,8 @@ def sort_directive(klass, attr, order, options)
         arel = klass.arel_attribute(attr)
         if order
           arel = arel.lower if options.map(&:downcase).include?("ignore_case")
-          arel = arel.desc if order.downcase == "desc"
-          arel = arel.asc if order.downcase == "asc"
+          arel = arel.desc if %w[desc descending].include?(order.downcase)
+          arel = arel.asc if %w[asc ascending].include?(order.downcase)
```

Comment:
```suggestion
          arel = (sort_order == :ascending ? arel.asc : arel.desc) if order.present?
```

maybe the `if` needs to be a block

Reply:
since we are only going here if order is specified, I think we can get rid of the if on this line (and then obviously we don't need if in a block because it is going away)

Reply:
i don't believe you about using `sort_order` here, sorry